app/components/pool/details/pool-configuration.html

```diff
@@ -49,14 +49,25 @@
             <bl-text-property label="Version" [value]="decorator.virtualMachineConfiguration.imageReference.version"></bl-text-property>
             <bl-text-property label="Node agent sku id" [value]="decorator.virtualMachineConfiguration.nodeAgentSKUId"></bl-text-property>
             <bl-text-property *ngIf="decorator.virtualMachineConfiguration.windowsConfiguration" label="Enable automatic updates" [value]="decorator.virtualMachineConfiguration.windowsConfiguration.enableAutomaticUpdates"></bl-text-property>
+            <bl-text-property *ngIf="decorator.virtualMachineConfiguration.containerConfiguration" label="Container type" [value]="decorator.virtualMachineConfiguration.containerConfiguration.type"></bl-text-property>
+            <bl-text-property *ngIf="decorator.virtualMachineConfiguration.containerConfiguration" label="Container images" [value]="decorator.virtualMachineConfiguration.containerConfiguration.containerImageNames"></bl-text-property>
+            <bl-table-property label="Container registries" *ngIf="decorator.virtualMachineConfiguration.containerConfiguration?.containerRegistries?.length > 0">
+                <bl-table-property-header>
```

Comment:
could maybe add a containerConfig() get method in the component to make this is little simpler.